### PR TITLE
PipsPager: update navigation buttons tooltip placement

### PR DESCRIPTION
**Pull request by beervoley**
## Description

This PR updates tooltip placement to display it in correct spot based on orientation.
## Motivation and Context

Tooltip placement for navigation buttons was the same in all orientations. It was a problem because in vertical orientation tooltip for the NEXT page navigation button was covering the pager itself.
## How Has This Been Tested?

Manually.
## Screenshots (if appropriate):

Vertical orientation:
image
image

Horizontal orientation:
image
image

### CODE REVIEW

File: dev/PipsPager/PipsPager.xaml [hunk 1]
```diff
@@ -69,12 +69,14 @@
                                 <VisualState x:Name="HorizontalOrientationView">
                                     <VisualState.Setters>
                                         <Setter Target="RootPanel.Orientation" Value="Horizontal"/>
+					                    <Setter Target="PreviousPageButton.(ToolTipService.Placement)" Value="Left"/>
```
Comment (ranjeshj) [inline dev/PipsPager/PipsPager.xaml:72]: nit: replace tabs with spaces

**Review by ranjeshj — APPROVED** [review]

**Comment by kmahone** [comment]
The build error that was hit by the Pipeline build has been fixed in main. If you merge the latest changes from main into your branch, this PR should get unblocked.

**Pull request merged**